src/theme/protocolTheme.ts

```diff
@@ -0,0 +1,29 @@
+// Style overrides for typography to match Mozilla Protocol design system
+// https://protocol.mozilla.org/
+import { useMemo, useState } from 'react';
+
+import { createTheme, Theme } from '@mui/material/styles';
+
+import components from './components';
+import typography from './typography';
+
+const useProtocolTheme = () => {
+  const [mode, setMode] = useState<'light' | 'dark'>('light');
```

Comment:
this seemed like the easiest/most logical place to put this hook. the example in the docs I believe creates a new component and wraps the `App` component. I think this approach is cleaner.